tests/capiTest/CMakeLists.txt

```diff
@@ -33,6 +33,9 @@ target_link_libraries(SchedulerCapiTest PRIVATE GTest::gtest GTest::gtest_main P
 
 # Generate paths Include for each config
 # Allows for easy use inside IDE
+
+string(REPLACE "\\" "/" Python3_STDLIB_PATH_C ${Python3_STDLIB})
```

Comment:
This looks odd, like the whole generation of the header file. These paths should be settable through the environment in which the test executable runs.